app/controllers/course/forum/topics_controller.rb

```diff
@@ -9,8 +9,9 @@ class Course::Forum::TopicsController < Course::Forum::ComponentController
   before_action :add_topic_breadcrumb
 
   def show
-    @posts = @topic.posts
-    @topic.views.create(user: current_user)
+    @posts = @topic.posts.order(created_at: :asc).includes(:creator).to_a
+    @topic.viewed_by(current_user)
+    @post = @topic.posts.build(title: t('course.discussion.posts.reply_title', title: @topic.title))
```

Comment:
this should be called reply_post or something like that, the controller is a topic controller, not posts controller